index.d.ts

```diff
@@ -7,6 +7,25 @@ declare namespace cliTruncate {
 		*/
 		readonly position?: 'start' | 'middle' | 'end';
 
+		/**
+		Truncate at different positions.
+
+		@default false
+
+		@example
+		```
+		cliTruncate('unicorns rainbow dragons', 20, {position: 'start', preferTruncationOnSpace: true});
+		//=> '…rainbow dragons'
+
+		cliTruncate('unicorns rainbow dragons', 20, {position: 'middle', preferTruncationOnSpace: true});
+		//=> 'unicorns…dragons'
+
+		cliTruncate('unicorns rainbow dragons', 6, {position: 'end', preferTruncationOnSpace: true});
+		//=> 'unico…'
+		````
+		*/
+		readonly preferTruncationOnSpace?: boolean;
```

Comment:
I think it should be moved to after the `space` option. That applies to all the other places it's mentioned too, like the readme.

Reply:
okay